### DISCUSSION TITLE
What are the new innovations in this whisper model?

### DISCUSSION THREAD

**Discussion by kafan1986** [openai/whisper#486] [Q&A]
As per my understanding the approach used by whisper is unique in preprocessing the training data and incorporating all components of an ASR pipeline like diarization LM etc. in a single end to end model. The loss is cross entropy which is usually not better than CTC and definitely not better than RNNT, assuming an ASR without a separate LM. Other model components are also not new like transformers etc., so apart from throwing in huge volume of training data used for training this model, how does this model achieves claimed great results? What major innovations am I missing?

**Comment by eisenzopf** [comment]
I've been evaluating the WER against very challenging call recordings that suffer from background noise and low quality encoding. Whisper performs better than any other speech to text engine that I've evaluated. Therefore, I would say that the innovation that Whisper offers is that it is a general domain, speaker indepdendent speech recognition engine that performs better than any other ASR on the market with no tuning.

**Reply by kafan1986** [reply]
@eisenzopf That I understand, even I have tested the whisper ASR engine and its accuracy is good. The question is how it is reaching this accuracy? Is it because of sheer amount of training data or is there something in the architecture of the neural network.

**Comment by FurkanGozukara** [comment]
More data , free and better than google premium API

data is king. have you seen any model trained with 680k hours of data?

for more data processing, you need more hardware.

if there someone else uses even inferior algorithm with 6.8m hours of data, it probably would surpass whisper in every way
